webpack/components/extensions/HostDetails/Tabs/DebsTab/DebsTab.js

```diff
@@ -0,0 +1,597 @@
+import React, { useCallback, useState, useRef } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  ActionList,
+  ActionListItem,
+  Dropdown,
+  DropdownItem,
+  DropdownSeparator,
+  DropdownToggle,
+  DropdownToggleAction,
+  KebabToggle,
+  Select,
+  SelectOption,
+  SelectVariant,
+  Skeleton,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import { TableVariant, Thead, Tbody, Tr, Th, Td, TableText } from '@patternfly/react-table';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import SelectableDropdown from '../../../../SelectableDropdown';
+import TableWrapper from '../../../../../components/Table/TableWrapper';
+import { useBulkSelect, useTableSort, useUrlParams, useSet } from '../../../../../components/Table/TableHooks';
```

Comment:
After https://github.com/Katello/katello/pull/10783 is merged you'll have to import these from `foremanReact`, just fyi

Reply:
Thanks, applied!